packages/data/lib/di/data_injector.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:data/dio/dio_builder.dart';
+import 'package:data/repository/network_repository.dart';
+import 'package:data/service/api_service.dart';
+import 'package:dio/dio.dart';
+import 'package:domain/use_cases/repository/network_repository.dart';
+import 'package:get_it/get_it.dart';
+
+Future<void> initDataModule() async {
+  _initDioModule();
+  _initApiServiceModule();
+  _initCancelToken();
+  _initRepositoryModule();
+}
+
+void _initApiServiceModule() {
+  GetIt.I.registerSingleton(
+      ApiService(GetIt.I.get(instanceName: 'palindrome_url')));
+}
```

Comment:
formatting. flutter formatting depends on commas. for ex, if u'll add comma before the last closing bracket, studio/vs code will format in the following way:

```
GetIt.I.registerSingleton(
        ApiService(GetIt.I.get(instanceName: 'palindrome_url')),
);
```

Play with it

Reply:
fixed